Package.swift

```diff
@@ -0,0 +1,20 @@
+// swift-tools-version:5.1
+import PackageDescription
+
+let package = Package(
+    name: "Mimus",
+    products: [
+        .library(
+            name: "Mimus",
+            targets: ["Mimus"]),
+    ],
+    dependencies: [],
+    targets: [
+       .target(
+            name: "Mimus",
+            dependencies: []),
+        .testTarget(
```

Comment:
There's also a test target `MimusExamples` which has some tests there (although more of an example rather than actual tests, it's all tested in `MimusTests` target), perhaps we should add that here too?